src/Stryker.CLI/Stryker.CLI/CLIOptions.cs

```diff
@@ -86,6 +86,14 @@ public static class CLIOptions
             ArgumentDescription = @"Used for matching the project references when finding the project to mutate. Example: ""ExampleProject.csproj""",
             JsonKey = "project-file"
         };
+       
+        public static readonly CLIOption<string> TestProjectFileName = new CLIOption<string>
```

Comment:
I'm not sure if this should be the exact name or a filter like the `project-file` param. For the project to mutate we filter on the name passed by the user. It could for example be:

```
--project-file "myProject"
```
if the project file is `C:/myproject/myproject.csproj`

But the following would also be accepted:
```
--project-file "myproject/myProject"
```
which gives user a less strict experience.

@Mobrockers what do you think? Should this be the exact name or a filter?

Reply:
I would prefer a filter, as it is implemented currently.